Panic while TestCryptoAes

machine 1 output:
```
=== RUN   TestCryptoAes
--- PASS: TestCryptoAes (0.86s)
```

machine 2 output:
```
=== RUN   TestCryptoAes
panic: A test was taking too long

goroutine 66 [running]:
github.com/robertkrimen/otto.tt.func1()
	/tmp/autopkgtest-lxc.zb1zh8ok/downtmp/autopkgtest_tmp/obj-x86_64-linux-gnu/src/github.com/robertkrimen/otto/testing_test.go:15 +0x3c
created by time.goFunc
	/usr/lib/go-1.11/src/time/sleep.go:172 +0x44
FAIL	github.com/robertkrimen/otto	2.241s
```

Some machines got PASS while TestCryptoAes, but some got FAIL.

I tried again twice in slow machine, and the result here:

First try:
```
=== RUN   TestCryptoAes
--- PASS: TestCryptoAes (1.93s)
```

Second try:
```
=== RUN   TestCryptoAes
panic: A test was taking too long

goroutine 34 [running]:
github.com/robertkrimen/otto.tt.func1()
	/root/go-team/micro/golang-github-robertkrimen-otto/golang-github-robertkrimen-otto-0.0~git20180617.15f95af/obj-x86_64-linux-gnu/src/github.com/robertkrimen/otto/testing_test.go:15 +0x3c
created by time.goFunc
	/usr/lib/go-1.11/src/time/sleep.go:172 +0x44
FAIL	github.com/robertkrimen/otto	2.161s
```

Extending the testing timeout is needed for slow computing environment?

--- a/testing_test.go
+++ b/testing_test.go
@@ -11,7 +11,7 @@ import (
 
 func tt(t *testing.T, arguments ...func()) {
 	halt := errors.New("A test was taking too long")
-	timer := time.AfterFunc(2*time.Second, func() {
+	timer := time.AfterFunc(20*time.Second, func() {
 		panic(halt)
 	})
 	defer func() {

